cpp/include/nvstrings/NVStrings.h

```diff
@@ -425,6 +425,16 @@ class NVStrings
      * @return The number of trues.
      */
     unsigned int is_empty( bool* results, bool devmem=true );
+    /**
+     * @brief Fills in matrix with code point values (integers) for each string.
+     *
```

Comment:
Maybe explain what a code point is. (First I've heard of it) Perhaps say that it's UTF-8 like in the python docstring.

Reply:
> Couple of suggestions. I would prefer that the function returns its results rather than taking a preallocated pointer. But I realize that nvstrings APIs may not work that way and may be redesigned in the future...

Yes, the entire NVStrings API relies on non-strings memory to be allocated by the caller so it could be placed in a cudf column without making a copy. This will be fixed with cudf strings column work where a column is returned instead.